discretize/mixins/omfModule.py

```diff
@@ -0,0 +1,115 @@
+"""
+A class for converting ``discretize`` meshes to OMF objects
+"""
+
+import omf
+import numpy as np
+
+
+def ravelDataArray(arr, mesh):
+    """Ravel's a numpy array into proper order for passing to the OMF
+    specification from ``discretize``/UBC formats
+    """
+    dim = (mesh.nCz, mesh.nCy, mesh.nCx)
+    return np.reshape(arr, dim).ravel(order='F')
+
+
+class omfInterface(object):
```

Comment:
Similar here, do you think a name of `OMFInterface` works? or is that too clumsy?

Reply:
Either is fine with me - I made it `omfInterface` to be consistent with `vtkInterface`

Reply:
I change this to `InterfaceOMF` as well as changed `vtkInterface` to `InterfaceVTK`

This makes more sense to me, is a bit more pep8 friendly, and starts a naming convention for more interface type mixins that will make finding via tab-complete a bit easier